autojudge/settings_production.py

```diff
@@ -0,0 +1,18 @@
+# To use these settings during development, run with
+# python manage.py runserver --settings=autojudge.settings_production
+
+import os
+from typing import List
+from .settings import *
```

Comment:
Can you add a comment like this below?

`# noqa: F403`

You can look at settings.py or submission_watcher_saver.py for reference.